### ISSUE TITLE
Display View Counts for all sections

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-feature-requests#607]
All view counts should be actual numbers which make it more active and alive instead of the current. 1k+, 2k+ etc... which psychologically make it less appealing and dead. See issue here: https://github.com/PHPfox-Official/phpfox-v4-issues/issues/2185

**Comment by spirogg** [comment]
Yes this is good idea. I second the motion to make this feature available for the actual number to show.

**Comment by dawn2323** [comment]
I Agree too!!
